src/jit/codegenarm64.cpp

```diff
@@ -2470,7 +2470,8 @@ CodeGen::genCodeForTreeNode(GenTreePtr treeNode)
                 // We unconditionally throw a divide by zero exception
                 genJumpToThrowHlpBlk(EJ_jmp, SCK_DIV_BY_ZERO);
 
-                // We still need to call genProduceReg
+                // We still need to call genConsumeOperands/genProduceReg for liveness update.
+                genConsumeOperands(treeNode->AsOp());
```

Comment:
@CarolEidt - arm64 has implemented its own genCodeForBinary() in codegenarm64.cpp and it is called from 4 places - out of which 3 of the uses are in GT_DIV/UDIV case.  Arm64 version of genCodeForBinary() is a simple wrapper method that gets the ins for oper and calls genConsumeOperands() followed by emitInsTernary().  For that reason, I have suggested to move out genConsumeOperands() responsibility to callers of genCodeForBinary().